stock/templates/registered/srpfleetdata.html

```diff
@@ -55,6 +57,8 @@ <h1 class="page-header text-center">SRP Fleet Data
                                target="_blank">{{ srpfleetrequest.killboard_link }}</a>
                         </td>
                         <td class="text-center">{{ srpfleetrequest.additional_info }}</td>
+			<td class="text-center">{{ srpfleetrequest.srp_ship_name }}</td>
```

Comment:
as I said before and I will say it again, my eyes read spaces. Formatting is important.